packages/back-end/src/init/config.ts

```diff
@@ -205,3 +212,21 @@ export function getConfigOrganizationSettings(): OrganizationSettings {
   reloadConfigIfNeeded();
   return config?.organization?.settings || {};
 }
+
+export function getConfigSegments(organization: string): SegmentInterface[] {
```

Comment:
We should add this to the documentation config.mdx page - https://docs.growthbook.io/self-host/config

Specific changes:
1. Intro paragraph, change to `and optionally dimensions and segments`
2. Add an example segment to the yml
3. Create a new `### Segments` section at the bottom underneath `### Dimensions`
4. In the Organization Settings section, change `In addition to data sources, metrics, and dimensions` to `In addition to the above`